src/nrc/utils/webtest.py

```diff
@@ -0,0 +1,7 @@
+from django_webtest import WebTest as DjangoWebTest
+from maykin_2fa.test import disable_admin_mfa
+
+
+@disable_admin_mfa()
+class WebTest(DjangoWebTest):
+    pass
```

Comment:
I don't see that much of an advantage to make this a separate class, let's just add the decorator to the testcases that need it instead of adding it everywhere